README.md

```diff
@@ -25,9 +42,27 @@ $ virtualenv venv
 $ source venv\bin\activate
 $ pip3 install --upgrade google-api-python-client google-cloud-speech google-cloud-texttospeech EbookLib bs4 xml-cleaner pydub
 ```
+## Offline TTS
+Set up virutalenv and install libraries for:
+* TensorFlow
+* ePub parsing
 
-#### Google Cloud Platform
-Set up a Google Cloud Platform account, following the instructions here: 
+```sh
+$ virtualenv venv
+$ source venv/bin/activate
+$ pip3 install -r requirements.txt
+```
+### Offline model downloads
+**Option 1: Download pre-zipped models**
+
+Download [the zip file](https://drive.google.com/file/d/1fcc8Rdl9WemkwFoiLg4p22vilwd5h8KV/view?usp=sharing) and extract to this directory.

```

Comment:
Can you clarify this? Do you mean the root directory of the repo?

Reply:
Yes you're right it is in fact repo root- added that.